recipes/boost-ext-ut/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
```

Comment:
not used

```suggestion
```

Reply:
Of course, it wouldn't be a proper PR without an unused import. Nice catch!